modules/ROOT/examples/live-demos/autocorrect/index.html

```diff
@@ -0,0 +1,10 @@
+<div id="tiny-ui">
+    <textarea id="autocorrect" style="width: 100%; height: 500px;">
```

Comment:
As I told @MitchCTiny via DMs these examples don't follow the correct code style. They should be using 2 spaces, not 4.